codec-http2/src/test/java/io/netty/handler/codec/http2/InboundHttp2ToHttpAdapterTest.java

```diff
@@ -607,6 +608,9 @@ public void run() throws Http2Exception {
             verify(serverListener, times(2)).messageReceived(requestCaptor.capture());
             capturedRequests = requestCaptor.getAllValues();
             assertEquals(2, capturedRequests.size());
+            // We expect to not have this header i nthe captured request so remove it now.
+            assertNotNull(request.headers().remove("x-http2-stream-weight"));
```

Comment:
Note this change was needed as we not correctly copied the headers before in our replace(...) methods.